src/Artsy/Router/buildServerApp.tsx

```diff
@@ -81,23 +83,34 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
           )
         }
 
-        const { loadableState, relayData: _relayData, styleTags } = await trace(
+        const {
+          loadableState,
+          relayData: _relayData,
+          styleTags,
+          bodyHTML,
+        } = await trace(
           "buildServerApp.fetch",
           (async () => {
             // Kick off relay requests to prime cache. TODO: Remove the need to
             // do this by using persisted queries.
-            ReactDOMServer.renderToString(<ServerApp />)
+            ReactDOMServer.renderToString(sheet.collectStyles(<ServerApp />))
             // Extract render queue for bundle split components using dyanamic `import()`
-            const state = await getLoadableState(<ServerApp />)
+            const state = null // await getLoadableState(<ServerApp />)
```

Comment:
We have yet to use dynamic imports / bundle splitting so maybe this can be removed until we better communicate the purpose. (Also, would like to follow up on how this relates to latest React.)

Reply:
cc @zephraph

Reply:
I was going to ask that, thanks 👍